presto-main/src/main/java/com/facebook/presto/cost/TableScanStatsRule.java

```diff
@@ -71,8 +70,7 @@ protected Optional<PlanNodeStatsEstimate> doCalculate(TableScanNode node, StatsP
                 .setOutputRowCount(tableStatistics.getRowCount().getValue())
                 .setTotalSize(tableStatistics.getTotalSize().getValue())
 
-                // TODO Handle the confidence level properly when filters are pushed into the tablescan
-                .setConfidence(FACT)
+                // TODO Handle the confidence level as fact when there are no filters
```

Comment:
It makes sense. We don't have reliable stats for scans even when there are no filters. So leaving it as default (low) is good.